app/lib/common/dialogs/nuke_confirmation.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:acter/features/onboarding/providers/onboarding_providers.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+
+// Can be extended to be reusable dialog as riverpod states get added.
+// Ref can be used to read any provider which are declared.
+void nukeConfirmationDialog(BuildContext context, WidgetRef ref) {
+  showDialog(
+    context: context,
+    builder: (BuildContext ctx) {
+      return AlertDialog(
+        title: const Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Icon(Atlas.bomb_bold,color: Colors.red,),
```

Comment:
I would also be not be in favor of using this icon, could be brush or some more friendly warning sign...

Reply:
brush?

It was intended to be something that no one clicks without understanding the risk ... as this is a super-wipe-out without backups ... and see comments as above ...

Reply:
As in means of cleaning as we can see some cleaning data apps, anyways I would like to hear what's the general opinion on this via feedback.